static/panes/cfg-view.js

```diff
@@ -102,25 +102,29 @@ function Cfg(hub, container, state) {
     this._editorid = state.editorid;
     this._binaryFilter = false;
 
-    this.functionPicker = $(this.domRoot).find('.function-picker').selectize({
+    var self = this;
```

Comment:
In other code we've tried not to rely on `self` instead of `this`. It's too easy to forget and accidentally use `this` in the function. Hopefully "soon" we'll transcode client JS from ES6, and then we can see the back of all this!

So; mildly prefer you kept the code using `bind`